src/sass/card/card.scss

```diff
@@ -0,0 +1,135 @@
+@import "../mixins/private/bubble-mixins";
+@import "../mixins/private/token-mixins";
+@import "../variables/variables";
+
+.card {
+    display: flex;
+    flex-direction: column;
+    width: $card-size-default;
+}
+
+.card__container {
+    position: relative;
+    height: $card-size-default;
+}
+
+@media screen and (max-width: $_screen-size-MD) {
+    .card {
+        width: $card-size-large;
+
+        .card__container {
+            height: $card-size-large;
+        }
+    }
+}
+
+.card__content {
+    display: flex;
+    height: inherit;
+    width: inherit;
+    justify-content: center;
+    align-items: center;
+    box-sizing: border-box;
+    background-color: var(--color-background-secondary, #f7f7f7);
+    border-radius: var(--border-radius-100, 16px);
+    padding: var(--spacing-150, 12px);
+}
+
+.card__content .progress-bar {
+    width: 100%;
+}
+
+.card__content--image {
+    background-size: contain;
+    background-repeat: no-repeat;
+    background-position: center;
+}
+
+.card__content--overflow {
+    opacity: 0.5;
+}
+
+.card .card__action {
+    @include bubble-close();
+
+    position: absolute;
```

Comment:
Why do we need position absolute. We should be able to put these inline I assume and style it that way.